src/Uno.UI/UI/Xaml/Media/AcrylicBrush/Android/AcrylicBrush.Rendering.Blur.Android.cs

```diff
@@ -0,0 +1,272 @@
+		/// A suggestion would be to set it to false for debug, and to true for releases.
+		/// </summary>
+		public static bool ThrowStopExceptionOnDraw { get; set; } = false;
+
+		private bool IsAndroidBlurPropertySet => AndroidBlurRadius > 0;
+
+		private double CurrentBlurRadius =>
+			IsAndroidBlurPropertySet ? AndroidBlurRadius : StyledBlurRadius;
+
+		//TODO
+		//protected void OnAttachedToWindow()
+		//{
+		//	if (MaterialFrame.AndroidBlurRootElement != null && _blurRootView == null)
+		//	{
+		//		UpdateAndroidBlurRootElement();
+		//	}
+		//}
+
+		//TODO
+		//protected void OnSizeChanged(int w, int h, int oldw, int oldh)
+		//{
+		//	base.OnSizeChanged(w, h, oldw, oldh);
+
+		//	LayoutBlurView();
+		//}
+
+		private void LayoutBlurView(ViewGroup view)
+		{
+			if (view.MeasuredWidth == 0 || view.MeasuredHeight == 0 || _realtimeBlurView == null)
+			{
+				return;
+			}
+
+			int width = view.MeasuredWidth;
+			int height = view.MeasuredHeight;
+
+			_realtimeBlurView.Measure(width, height);
+			_realtimeBlurView.Layout(0, 0, width, height);
+		}
+
+		private void DestroyBlur(BindableView view)
+		{
+			if (!_realtimeBlurView.IsNullOrDisposed())
+			{
+				view.RemoveView(_realtimeBlurView);
+			}
+
+			_realtimeBlurView?.Destroy();
+			_realtimeBlurView = null;
+		}
+
+		//private void UpdateAndroidBlurRootElement()
+		//{
+		//	if (MaterialFrame.AndroidBlurRootElement == null)
+		//	{
+		//		return;
+		//	}
+
+		//	var formsView = MaterialFrame.AndroidBlurRootElement;
+		//	var renderer = Platform.GetRenderer(formsView);
+		//	if (renderer == null)
+		//	{
+		//		return;
+		//	}
+
+		//	bool IsAncestor(Element child, Layout parent)
+		//	{
+		//		if (child.Parent == null)
+		//		{
+		//			return false;
+		//		}
+
+		//		if (child.Parent == parent)
+		//		{
+		//			return true;
+		//		}
+
+		//		return IsAncestor(child.Parent, parent);
+		//	}
+
+		//	if (!IsAncestor(MaterialFrame, MaterialFrame.AndroidBlurRootElement))
+		//	{
+		//		throw new InvalidOperationException(
+		//			"The AndroidBlurRootElement of the MaterialFrame should be an ancestor of the MaterialFrame.");
+		//	}
+
+		//	Platform.SetRenderer(formsView, renderer);
+		//	_blurRootView = renderer.View;
+
+		//	_realtimeBlurView?.SetRootView(_blurRootView);
+		//}
+
+		private void UpdateAndroidBlurOverlayColor(bool invalidate = true)
+		{
+			if (IsAndroidBlurPropertySet)
+			{
+				var tintColorWithOpacity =
+					Color.FromArgb(
+						(byte)(TintOpacity * TintColor.A),
+						TintColor.R,
+						TintColor.G,
+						TintColor.B);
+				Android.Graphics.Color tintColor = tintColorWithOpacity;
+				_realtimeBlurView?.SetOverlayColor(tintColor, invalidate);
+			}
+		}
+
+		private void UpdateAndroidBlurRadius(bool invalidate = true)
+		{
+			if (IsAndroidBlurPropertySet)
+			{
+				//TODO
+				//_realtimeBlurView?.SetBlurRadius(
+				//	ContextHelper.Current.ToPixels(AndroidBlurRadius),
+				//	invalidate);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932087)